### ISSUE TITLE
Minimum System Requirements for Outreachy

### ISSUE THREAD

**Issue by germonprez** [chaoss/governance#156]
Hi everyone, 

As part of getting the projects available for Outreachy, I need a few more things from you. On the following page, I think I have most information: 

https://github.com/chaoss/governance/blob/master/Outreachy-ideas.md

But a few things that I need you to spell out for me. You can just include answers in this Issue. 

1) Minimum System Requirements. Please remember that for Outreachy, participants often have old computers. For example,
Internet access
Single Core 1.0 Ghz
64Mb Graphics Card
1Gb of Ram
40Gb Hard drive,
Lightweight Linux Operating System (Windows and Mac are fine too)

2) Links to getting started tutorials or documentation, how applicants can find contribution tasks on your project website or issue tracker, who they should ask for tasks, and everything they need to know to get started.

@sgoggins @GeorgLink @Polaris000 @sduenas @valeriocos @zhquan @alpgarcia @dlumbrer @samanthavenialogan

**Comment by germonprez** [comment]
I'll add that I need to get this information posted by Feb 24, 2020.

**Comment by GeorgLink** [comment]
I saw this issue and hope that the maintainers closer to the technology can provide this information.

**Comment by samanthavenialogan** [comment]
So do you need me to help with this then or?

Call me for your free consultation! 307 274 5516

**Comment by germonprez** [comment]
Update on this. I heard from Sage that the mentors have to submit the proposals/projects to Outreachy/CHAOSS because they have to sign the mentor agreement. I won't be able to submit on your behalf. 

It makes me think that maybe we should scale back the number of projects to 4 or so. We have 8 at the moment.

**Comment by GeorgLink** [comment]
I think having only 4 is manageable for students as well. What is the
process for mentors to submit the proposals?

**Comment by samanthavenialogan** [comment]
We me know what dylan and I need to do and well get it to you Asap.  :) if
you're reducing projects how do you plan to choose?

Call me for your free consultation! 307 274 5516

**Comment by GeorgLink** [comment]
Good question @samanthavenialogan, I would choose projects so that we have a broad range of topics and mentors. I suggest dropping the most similar options.

**Comment by germonprez** [comment]
Can you see this page: https://www.outreachy.org/communities/cfp/chaoss/

**Comment by GeorgLink** [comment]
Yes, I can see that page. Thanks!

**Comment by germonprez** [comment]
Great. I reached out to @sgoggins so he will get two in from Augur

**Comment by valeriocos** [comment]
Hi @germonprez , I have just talked with @GeorgLink, the idea is to submit the following ideas:
- https://github.com/chaoss/grimoirelab/issues/288
- https://github.com/chaoss/grimoirelab/issues/286

@samanthavenialogan can you submit the form for outreachy for https://github.com/chaoss/grimoirelab/issues/288? thanks

**Comment by germonprez** [comment]
Hi @valeriocos this sounds great! Thanks for doing this and let me know if you have trouble getting these submitted by Feb 24: https://www.outreachy.org/communities/cfp/chaoss/

I'd do it for you but I can't as it requires a mentor sign-off by you :)

**Comment by samanthavenialogan** [comment]
Yes. I will do this later tonight once I'm home from the conference.

**Comment by samanthavenialogan** [comment]
Hey do any of you have time to go through this with me by chance? there's a lot of content on this mentorship application I imply don't know such as this question: 

How long has your team been accepting publicly submitted contributions?

or this one: 
How can applicants make a contribution to your project?

**Comment by valeriocos** [comment]
Hi @samanthavenialogan, sorry for the late reply!

> How long has your team been accepting publicly submitted contributions?

More than 2 years

> How many regular contributors does your team have?

3-5 people

> Title and long description

This info is similar to the issue https://github.com/chaoss/grimoirelab/issues/288

> Minimum system requirements:

I would say: Hardware: 2 CPUs, 8GB memory RAM, Software: Linux OS (possibly Ubuntu). These requirements are taken from https://github.com/chaoss/grimoirelab#getting-started

> How can applicants make a contribution to your project?

I drafted the content below for the idea at https://github.com/chaoss/grimoirelab/issues/286, feel free to adapt it for social currency idea:
```
Applicants willing to contribute to this project should introduce themselves by commenting the issue https://github.com/chaoss/grimoirelab/issues/286.

Contributions will follow the classical issue/pull request workflow. Thus, an applicant will open an issue on https://github.com/chaoss/grimoirelab describing the problem and how to solve it. After evaluating the proposal with the mentors (directly on the issue or via instant messaging channels), the applicant will submit one or more pull requests to the repositories hosting the components to be modified. The pull requests will go through a code review process. In case of a positive review, the pull request will be merged to the code base. 

The issue https://github.com/chaoss/grimoirelab/issues/286 contains a list of microtasks to get started with the project. Applicants should create a repository in their GitHub profiles and address each task. At any time, they can ask for help or feedback to the mentors (using the mention mechanism provided by GitHub).

The tutorial and documentation is available at https://chaoss.github.io/grimoirelab-tutorial/ and https://github.com/chaoss/grimoirelab#getting-started. Any further clarification can be asked by commenting the issue https://github.com/chaoss/grimoirelab/issues/286 (and using the mention mechanism). 
```

> Repository:

https://github.com/chaoss/grimoirelab

> Issue tracker:

https://github.com/chaoss/grimoirelab/issues

> Newcomer issue tag:

We don't have newcomer issue tags

> Intern tasks

They are the microtasks listed at https://github.com/chaoss/grimoirelab/issues/288

> Intern benefits:

I drafted the content below for https://github.com/chaoss/grimoirelab/issues/286, feel free to reuse/adapt it
```
The intern will improve his knowledge of ETL processes with Python, ElasticSearch and Kibana. Thus, at the end of the program he will be able to load, transform and visualize data using popular technologies.
```

> Community benefits:

This is up to you :)

**Comment by germonprez** [comment]
Thanks @valeriocos -- first Outreachy project approved: 

https://www.outreachy.org/apply/project-selection/#chaoss

@sgoggins as you have time this weekend, please upload a project or two :)

**Comment by valeriocos** [comment]
> https://github.com/chaoss/governance/issues/156#issuecomment-589895308

@samanthavenialogan how is it going with the submission? Do you need some support?

**Comment by samanthavenialogan** [comment]
Hey all,

Sorry for the late reply on this.  I was busy this weekend with a
conference for Keystone Symposia and it's taken a lot of my attention.  The
meeting season dies down here in a week though :)

Regarding this project, I did manage to submit.  I'm not too sure I did all
that great a job and I hope you can edit it but it looks like it submitted
successfully :)

I also sent @Dylan Marcy <dylan@sociallyconstructed.online>  a request to
sign a co-mentorship design. I'm not sure if the other mentors for the SCMS
will need to do so as well?

Samantha V. Logan
RESCQU NET (TYC)
samantha@rescqu.net
Samantha@transyouthchannel.org
307-274-5516

**Comment by shreyaprasad41** [comment]
Hey @samanthavenialogan I am an outreachy aspirant and want to contribute to CHAOSS. Can you please guide me through the procedure for contribution in CHAOSS?

**Comment by samanthavenialogan** [comment]
> Hey @samanthavenialogan I am an Outreachy aspirant and want to contribute to CHAOSS. Can you please guide me through the procedure for contribution in CHAOSS?

Hey Shreya! 

This is exciting! I'm actually pretty new here as well, both as a mentor and to GitHub, but we'd love to help you get started!  

Two really good points of contact in pretty much anything GSOC/Outreachy related is @germonprez and @GeorgLink.  

Here are a few good links you may be interested in following to get started: 
1. Get to know [CHAOSS](https://chaoss.community/participate/), 
2. [install GrimoireLab](https://github.com/chaoss/grimoirelab), 
3. Do some preliminary research on what the projects are available for GSOC/Outreachy [here](https://github.com/chaoss/governance/blob/master/Outreachy-ideas.md)
4. Flag yourself as interested for an outreachy project [here](https://github.com/chaoss/governance/blob/master/Outreachy-interest.md) by completing a micro-task for that event and then posting it on the page as instructed, 
5. and ultimately [join the mailing list](https://lists.linuxfoundation.org/mailman/listinfo/chaoss) where you can introduce yourself and begin to communicate with us all :) 

We already have 3 GSOC/Outreachy people the mailing list this week who have taken up an interest and it's a pretty solid community of people willing to help or even just talk ^^.  When you find a place to specialize, we'll point you toward the sub-communities most available.

I should mention that my co-founder, Dylan, and I are mentoring along with @valeriocos and Georg but we are non-coding mentors, so georg and Matt are probably your best bet for questions on the projects in general. 

Otherwise, We'll be on the project about the [Social Currency Metrics system](https://chaoss.community/metric-social-currency-metric-system/) to be implemented via GrimoireLab. My role is to help with the non-coding aspect of the system in regards to qualitative data processing and the design of the system.

**Comment by miracle-E** [comment]
I'm an outreachy applicant and would love to contribute.

**Comment by GeorgLink** [comment]
Hi @miracle-E, welcome to CHAOSS! I suggest you familiarize yourself with the Outreachy application process and how CHAOSS does things: https://github.com/chaoss/governance/blob/master/Outreachy-interest.md
